src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -5,10 +5,53 @@
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static final int USER_MIN_AGE = 18;
+    private static final int PASSWORD_MIN_LENGTH = 6;
     private final StorageDao storageDao = new StorageDaoImpl();
 
     @Override
     public User register(User user) {
-        return null;
+        return checkUser(user) ? user : null;
     }
+
+    private boolean checkUser(User user) {
+        if (isValidUserLogin(user) && isValidUserAge(user) && isValidUserPassword(user)) {
+            storageDao.add(user);
+            return true;
+        }
+        return false;
+    }
```

Comment:
Why do you do this in method register ?

Reply:
One method should perform one task. Register method only registers. But I can mistake.